### DISCUSSION TITLE
Discussion Topics for Aug 16, 2021 Engineering Sync

### DISCUSSION THREAD

**Discussion by carlosfelix2** [CDCgov/prime-reportstream#1857] [Show and tell]
Please post your topics for discussion:
1. @carlosfelix2 - Performance measurement library that could be used to trend performance over time in RS

**Comment by td-usds** [comment]
- Continuous Deployment and the requirements therefore (test coverage, Quality Gates): we need to work with Engineering to come up with the right requirements here
    - What should the code coverage be be? Which tests contribute to Code Coverage numbers (unit tests, E2E tests, ...)
    - What other tests need to succeed and would make people comfortable with CD & what is the timeline to get these implemented?
        - Performance Tests with degradation QG?
    - Are there any other dependencies?

**Comment by MauriceReeves-usds** [comment]
I would like to discuss this: https://app.zenhub.com/workspaces/prime-data-hub-5ff4833beb3e08001a4cacae/issues/cdcgov/prime-reportstream/1866

**Comment by MauriceReeves-usds** [comment]
I would also like to discuss the CLIA conundrum that has come up recently.
